src/encoded/static/libs/ui/panel.js

```diff
@@ -264,6 +264,10 @@ TabPanel.defaultProps = {
 };
 
 
+// Check to see if child of tab has "disabled" class
+const checkIfDisabled = (input) => input.children && input.children.props && input.children.props.className && input.children.props.className.includes('disabled');
```

Comment:
Since this is an internal class, I assume you can at least trust that no one will give you a null "input" parameter to deal with.